cdap-docs/examples-manual/source/examples/wikipedia-data-pipeline.rst

```diff
@@ -101,21 +101,32 @@ Deploying the Example
 Since deploying the WikipediaPipelineApp involves loading an artifact and creating an application from it, the
 preferred method of deploying it is to use the CDAP CLI.
 
-- Load the Artifact::
+- Load the Artifact:
 
-    $ cdap-cli.sh load artifact target/WikipediaPipeline-3.3.0-SNAPSHOT.jar name WikipediaPipelineApp version
-    3.3.0-SNAPSHOT
-    Successfully added artifact with name 'WikipediaPipelineApp'
+  .. container:: highlight
+
+    .. parsed-literal::
+    
+      |$| cdap-cli.sh load artifact target/WikipediaPipeline-\ |release|.jar name WikipediaPipelineApp version |release|

```

Comment:
If you don't have a backslash, the parser does not see the start of the replacement: a replacement (two OR bars enclosing a label) always starts with whitespace. But we don't whitespace, so the backslash says to remove it when finished parsing. Does that explanation make sense?